JitsiConference.js

```diff
@@ -467,7 +468,8 @@ JitsiConference.prototype._init = function(options = {}) {
             preferredCodec: _getCodecMimeType(config.p2p?.preferredCodec),
             screenshareCodec: browser.isMobileDevice()
                 ? _getCodecMimeType(config.p2p?.mobileScreenshareCodec)
-                : _getCodecMimeType(config.p2p?.screenshareCodec)
+                : _getCodecMimeType(config.p2p?.screenshareCodec),
+            enableAV1ForFF: options.testing?.enableAV1ForFF
```

Comment:
Do we have to disable AV1 for p2p? Spatial scalability shouldn't matter for p2p case right?

Reply:
Isn't it the temporal scalability we are worried about?

Reply:
Done